src/Query.php

```diff
@@ -1300,6 +1300,27 @@ public static function create($from)
         ]);
     }
 
+    /**
+     * Constructor for query object
+     */
+    public function __construct($config = [])
+    {
```

Comment:
please use `yii\di\AbstractContainer::configure()` instead

Reply:
I've searching for some example, but I've not found nothing that use AbstractContainer.

I've tryied with:

        return \yii\di\AbstractContainer::configure( Query::class, $config);

but the first parameters has to be an object. 

I can't pass new Query() because I'm already inside the Query contructor

Reply:
Just use `$this` it is available in the constructor.
You can see example here: https://github.com/yiisoft/yii-core/blob/d87cecdb27b21773e75c08137fe644114823f8d8/src/validators/Validator.php#L197